python/tvm/driver/tvmc/compiler.py

```diff
@@ -59,6 +59,12 @@ def add_compile_parser(subparsers):
         default="",
         help="comma separarated list of formats to export, e.g. 'asm,ll,relay' ",
     )
+    parser.add_argument(
+        "--input-shape",
+        type=common.parse_input_shapes,
+        metavar="INPUT_SHAPE,[INPUT_SHAPE]...",
+        help="for PyTorch, e.g. '(1,3,224,224)'",
```

Comment:
Maybe clarify that it is in fact mandatory for PyTorch.

Reply:
Agree. It's confusing to see such a general option only for PyTorch. I would suggest the following changes:
1. Make `--input-shape` as a general option for all frontends. If present, we skip the input shape inference.
2. `--input-shape` is optional by default. However, if users want to process a PyTorch model but don't specify `--input-shape`, we throw out an error in the PyTorch frontend.